README.md

```diff
@@ -62,7 +62,9 @@ Display multiple files at once
 > bat src/*.rs
 ```
 
-Read from stdin, determine the syntax automatically
+Read from stdin, determine the syntax automatically (note, this will only work
+if the syntax can be determined from the initial line of the file, usually
```

Comment:
Small nitpick, but could we use "first line" instead of initial line? That terminology is more ubiquitious, and I think it would be easier for non-native speakers to understand.

Reply:
Yes that makes sense, I've updated that and also clarified a little more about "what" will only work if the syntax can be determined.